static/js/filter.js

```diff
@@ -216,6 +216,7 @@ function encodeOperand(operand) {
 }
 
 function decodeOperand(encoded, operator) {
+    encoded = encoded.replace(/"/g, '');
```

Comment:
It's not a perfect fix, but I'll replace `"` with `%22` in our encoding for the search bar.